Change the page size from 1920 * 1080 to 1280 * 720

W1uSeven7 opened the pull request:
Change the page size from 1920 * 1080 to 1280 * 720
This time should be fine.Thank you!
Steven - 2025.11.04

W1uSeven7 committed:
Change the page size from 1920 * 1080 to 1280 * 720

W1uSeven7 committed:
Merge branch 'main' into about-screen-design

W1uSeven7 committed:
Delete unnecessary diagonal lines

JacobLee23 reviewed (changes requested):
For the most part, these changes look good; I think only a few minor follow-up revisions are needed. In general, I think it would be good if the major components for this view are retained as class attributes instead of just constant local variables, but that's up to your discretion.

W1uSeven7 committed:
refactor(AboutScreen): separate concerns and improve code structure

- Move controller logic from View to Controller module

- Convert main components to class attributes for better maintainability

- Add BASE_WIDTH and BASE_HEIGHT constants to constants.ts

- Remove unused variable aliases (panelX, panelY)

- Add getCloseGroup() method to expose close button for event binding

W1uSeven7 committed:
Merge branch 'main' into about-screen-design

JacobLee23 reviewed (changes requested):
All the functionality and code organization looks much better. It looks like there were just a couple issues with rebasing/merging, but once those are resolved then these changes should be good to go.

W1uSeven7 committed:
Sync constants.ts with main branch

- Synced constants.ts with main branch
- Restored changes in main.ts
- Reverted Base-related parameters from constants.ts back to AboutView.ts

W1uSeven7 committed:
merge code on main branch

JacobLee23 commented:
The changes proposed for this pull request have already been approved. This branch needs to be rebased both locally and remotely before it can be merged.

W1uSeven7 committed:
combine newest main  branch

W1uSeven7 committed:
Add navigation logic for About button on Title page and exit button on About page

JacobLee23 commented:
What is the purpose of 12fc9eb46e7e9877ad022248a3bb39db01982ca7? This PR was already approved prior to the most recent pair of commits, it just needed you to merge it

W1uSeven7 commented:
> What is the purpose of [12fc9eb](https://github.com/CSE110-Coblenz/cse-110-project-team-6/commit/12fc9eb46e7e9877ad022248a3bb39db01982ca7)? This PR was already approved prior to the most recent pair of commits, it just needed you to merge it

Got it. When I pulled the latest code from the remote main branch to my local environment, I found that these two features had bugs. I’ll pull it again in a bit
